### DISCUSSION TITLE
How to set gpu render

### DISCUSSION THREAD

**Discussion by huangfuhao** [AvaloniaUI/Avalonia#9468] [Q&A] [answered]
When I  debug application it render by gpu, but it only render by  cpu after publish on windows platform , how do I set in order use gpu render after publish

**Comment by kekekeks** [comment, marked as answer]
Use `win7-x86`/`win7-x64` runtime identifier.

**Reply by huangfuhao** [reply]
It wroks, but create a new folder named win7-x64 not publish folder， It doesn't matter ,Thanks a lot
